### PR TITLE
[Feature] Add DarkMode toggle to User Settings

### PR DESCRIPTION
**Pull request by damassi**
Adds the ability to explicitly set dark mode via the user's settings page making things a bit more discoverable. 

Not sure how to [update the docs](https://support.are.na/help/is-there-a-dark-mode), though, which only mention the key combo. 

darkmode

### CODE REVIEW

**Comment by damassi** [comment]
One q I have is would it make sense to also place a toggle in the main shell or directly in the nav? When browsing through content its nice to be able to quickly toggle between things for a/b purposes because each mode makes different visual qualities pop out. That said being under settings does match the mobile app.

**Comment by broskoski** [comment]
I think it's fine to leave here now. I can imagine putting it in the main menu, but this is so straightforward and like you said, mirrors the mobile app.

File: src/v2/components/UserSettings/index.tsx [hunk 1]
```diff
@@ -480,6 +481,47 @@ const UserSettings: React.FC<UserSettingsProps> = ({ me, updateAccount }) => {
...
+
+              <Field
+                name="darkmode"
+                initialValue={document.documentElement.dataset.theme === 'dark'}
+              >
+                {props => {
+                  return (
+                    <>
+                      <InputContainer mt={6} mb={5}>
+                        <Label>Darkmode</Label>
```
Comment (broskoski) [inline src/v2/components/UserSettings/index.tsx]: So minor: this should be "Dark mode"

File: src/v2/components/UserSettings/index.tsx [hunk 2]
```diff
@@ -480,6 +481,47 @@ const UserSettings: React.FC<UserSettingsProps> = ({ me, updateAccount }) => {
...
+                          value={props.input.value}
+                          onSelect={value => {
+                            const theme = value === true ? 'dark' : 'default'
+                            toggleOrSetTheme(theme)
+                          }}
+                        >
+                          <RadioOptions.Option value={true}>
+                            {({ selected }) => (
+                              <Text f={4} mb={3} selected={selected}>
+                                Enable Darkmode
```
Comment (broskoski) [inline src/v2/components/UserSettings/index.tsx]: Also minor: change to just "Enable / Disable"

Comment (damassi) [reply src/v2/components/UserSettings/index.tsx]: I set it to be more verbose here based on the other toggles, which explicitly spell out each option. Assumed it was related to accessibility. Will fix 👍

**Pull request merged**